ui/public/i18n/en/translation.json

```diff
@@ -75,7 +75,12 @@
         "pecbridge_admin_mail_format": "Must be a valid email address",
         "phonebook_instances": "Synchronize Phonebook with NethVoice instance",
         "choose_phonebook_instance": "Choose a NethVoice instance in the list",
-        "no_phonebook_instance": "Disable Phonebook synchronization"
+        "no_phonebook_instance": "Disable Phonebook synchronization",
+        "lets_encrypt_tips": "This option enables automatic request of a Let's Encrypt TLS certificate for this application. You can manage the certificate afterward in Settings > TLS certificates.",
+        "cannot_obtain_certificate": "Cannot obtain a Let's Encrypt certificate for {hostname}",
+        "go_to_tls_certificates": "Go to TLS certificates",
+        "lets_encrypt_disabled_warning_description": "The certificate might be removed. The app may temporarily become unreachable until a valid certificate is restored",
```

Comment:
I'd change this to: "The application will be temporarily unavailable, and HTTP clients connected to {node} will experience a brief disconnection.".

Also, remember to pass the `node` variable the translation function